source/_patterns/00-atoms/forms/select.yaml

```diff
@@ -13,13 +13,28 @@ schema:
     name:
       type: string
       minLength: 1
-    state:
+    variant:
       type: string
       enum:
         - valid
         - error
+        - info
+    userInputInvalid:
```

Comment:
This is a helper for `state` == `error`? Seems ok given the limitations of Mustache. (Maybe call it `isError`? Or maybe `error` should be `invalid`, so `isInvalid`?)